content/webapp/pages/event.js

```diff
@@ -239,20 +240,25 @@ class EventPage extends Component<Props, State> {
               [spacing({s: 0, m: 2}, {margin: ['left']})]: true
             })}>
               {!event.isPast &&
-                <SecondaryLink
-                  link={{href: '#dates', as: '#dates'}}
-                  text={`See all dates`}
-                  icon={'arrowSmall'}
-                  trackingEvent={{
-                    category: 'component',
-                    action: 'date-times-jump-link:click',
-                    label: event.id
+                <NextLink
+                  href={`#dates`}
+                  as={`#dates`}>
+                  <a onClick={() => {
+                    trackEventV2({
+                      eventCategory: 'NextLink',
```

Comment:
See above comment.
https://github.com/wellcometrust/wellcomecollection.org/pull/3939/files#r246752581

Reply:
And maybe call this something like `EventDatesLink`?